utils/Canary/src/main/java/canary/mqtt5/Mqtt5Canary.java

```diff
@@ -364,8 +370,6 @@ public void accept(Mqtt5WebsocketHandshakeTransformArgs t) {
 
     public static void setupOperations() {
         // For now have everything evenly distributed
-        clientsOperationsList.add(CANARY_OPERATIONS.OPERATION_NULL);
-        clientsOperationsList.add(CANARY_OPERATIONS.OPERATION_START);
```

Comment:
Just curious as to why we're removing these.

Reply:
NULL:   The operation would do nothing, therefore I just remove it. 
START: I think it might be not necessary for a specific START operation, as we will always start the connection after the client stopped (There is a STOP operation in the list). If the client is stopped, then before the client does any other operation, it would start the client first.